src/FSharp.Formatting.ApiDocs/GenerateModel.fs

```diff
@@ -2264,8 +2264,11 @@ module internal SymbolReader =
         do
             replacedParagraphs
             |> Seq.collect collectParagraphIndirectLinks
-            |> Seq.filter (linkNotDefined doc)
-            |> Seq.map (getTypeLink ctx)
+            |> Seq.choose (fun line ->
+                if linkNotDefined doc line then
```

Comment:
Please invert the if/else here, I prefer the negative branch to be the if branch.

Reply:
I updated to use a local variable to prevent a double negative. Let me know if you prefer to not use this

Reply:
I feel like updating the definition of `linkNotDefined` to `linkDefined` might also be an option. It is only used once and could probably ne simplified to a `List.exist` instead of the map/reduce combo.